Changes

```diff
@@ -7,6 +7,12 @@
   Windows again (fault introduced by 4.06 Unicode changes).
   (David Allsopp)
 
+### Runtime system:
+
+- GPR#1752: do not alias function arguments to sigprocmask (Anil Madhavapeddy)
+
+- GPR#1753: avoid potential off-by-one overflow in debugger socket path
+  length (Anil Madhavapeddy)
```

Comment:
could you keep a blank line after this line?

Reply:
The spacing of the Changes entry is also arguably wrong, but it was also wrong in the original Changes entry. Would you mind moving the credit line to its own line in both cases?

Reply:
(The general format of Changes entry is documented in [CONTRIBUTING.md#changelog](https://github.com/ocaml/ocaml/blob/trunk/CONTRIBUTING.md#changelog))

Reply:
Sure, I took that entry as it was but I can edit it. I'll do that when I'm back to my computer!

Reply:
I also was unsure of whether to leave the original PR number, but I assumed it was the correct choice as the discussions took place there, do you confirm that choice?

Reply:
Leaving the original PRs is fine.
(It makes it very easy for people comparing the Changelogs to understand which entries have been moved around after the fact.)